public/mix-manifest.json

```diff
@@ -1,5 +1,10 @@
 {
     "/js/app.js": "/js/app.js",
     "/css/app.css": "/css/app.css",
+<<<<<<< HEAD
+    "/css/login-forget.css": "/css/login-forget.css",
```

Comment:
Please resolve conflict

Reply:
![FD9E0B6E-5E33-4FAC-B698-9C33DDE3D29E](https://github.com/kredo-school/12_banto_stock/assets/126224292/d2e78a6d-532d-4c42-86d8-d705962684fb)
Do you mean to remove these when I push?" /css/login-forget.css":"/css/login-forget.css",

Reply:
i mean this file mix-manifest.json